samples/java-spring-eventsourced-counter/src/it/java/com/example/TestkitConfig.java

```diff
@@ -11,11 +12,13 @@
 public class TestkitConfig {
     // end::class[]
 
-    @Profile("with-acls")
     // tag::acls[]
     @Bean
     public KalixTestKit.Settings settings() {
-        return KalixTestKit.Settings.DEFAULT.withAclEnabled(); // <1>
+        return KalixTestKit.Settings.DEFAULT.withAclEnabled()
+          .withMockedTopicSubscription("counter-commands")
+          .withMockedTopicDestination("counter-events")
+          .withMockedTopicDestination("counter-events-with-meta"); // <1>
```

Comment:
Here is the sample of testkit config to enable mocking eventing for selected parts of the application.